crml/staking/src/lib.rs

```diff
@@ -1364,15 +1283,15 @@ decl_module! {
 		}
 
 		/// Set the minimum bond amount.
-		#[weight = T::WeightInfo::set_minimum_bond()]
+		#[weight = 0]
```

Comment:
can you make this `DispatchClass::Operational` and give it some constant weight like `1_000_000`